grc/core/generator/Generator.py

```diff
@@ -246,6 +247,7 @@ def uses_var_id():
             'msgs': msgs,
             'generate_options': self._generate_options,
             'callbacks': callbacks,
+            'version': gr.version
```

Comment:
GRC has its own version field. Check-out grc/core/Config.py
This is, of course, the same as gr.version, but it serves a the single source within grc code.
Please use that instead